src/pytest_cov/engine.py

```diff
@@ -67,13 +67,16 @@ def _data_suffix(name):
 class CovController:
     """Base class for different plugin implementations."""
 
-    def __init__(self, cov_source, cov_report, cov_config, cov_append, cov_branch, config=None, nodeid=None):
+    cov: coverage.Coverage | None
+
+    def __init__(self, cov_source, cov_report, cov_config, cov_append, cov_branch, cov_precision, config=None, nodeid=None):
```

Comment:
It's probably best to make the new arg last + maybe kw-only so that it isn't a breaking change in the signature.

Reply:
Well now that you mention it, it does look odd doesn't it. Such a large number of positional always leads to some trouble. Funny thing, the config and nodeid are always passed. It was like that since the first commit where this was set to be a common lib for both pytest and nose: https://github.com/pytest-dev/pytest-cov/commit/7c6448ddf699c20323068ee7c2a17aa60fd3a3ae